src/Aspire.Dashboard/Components/Controls/ResourceDetails.razor

```diff
@@ -121,6 +121,22 @@
                               HighlightText="@_filter"
                               GridTemplateColumns="1fr 1.5fr" />
             </FluentAccordionItem>
+            <FluentAccordionItem Heading="@ControlStringsLoc[nameof(ControlsStrings.ResourceDetailsVolumesHeader)]" Expanded="true">
```

Comment:
This is only valid for containers, right? I think there should be an if around this section to only show for containers.